part2/Crypotology(Lab1)/src/main/java/lab1/FastModPow.java

```diff
@@ -0,0 +1,37 @@
+package lab1;
```

Comment:
use Java Code Convention for naming packages

Reply:
Done